package.json

```diff
@@ -128,7 +127,9 @@
     "argparse": "^1.0.10",
     "dexie": "^3.0.3",
     "dexie-export-import": "^1.0.0",
+    "eventemitter3": "^4.0.7",
     "inquirer": "^8.1.1",
-    "inquirer-fuzzy-path": "^2.3.0"
+    "inquirer-fuzzy-path": "^2.3.0",
+    "lodash": "^4.14.170"
```

Comment:
In fact the lodash is already in the `devDependencies`, I just move it into the `dependencies` that can be pulled into production build. If you are talking about the size that adding lodash size, I don't think it is even an issue, because that according to the doc, webpack should be smart enough to do the `tree-shaking` to bundle the imported function itself only, not the whole bunch of lodash.
And yes, I agree that `lodash` is another things to learn, although it is widely used nowadays, so that I have only added it into the "fundamental" code, and try to keep `job components` easier to read or use.